src/frontends/paddle/src/op/elementwise_ops.cpp

```diff
@@ -46,6 +46,10 @@ NamedOutputs elementwise_greater_equal(const NodeContext& node_context) {
     return elementwise_ops<default_opset::GreaterEqual>(node_context);
 }
 
+NamedOutputs elementwise_not_equal(const NodeContext& node_context) {
+    return elementwise_ops<default_opset::NotEqual>(node_context);
```

Comment:
1. OpenVINO NotEqual op return the type the same as input type while Paddle not_equal op return the bool type. The return type is different. Will it really work in a paddle model containing 'not_equal'?
2. paddle 'not_equal' has an optional input 'cond'. How to deal with the 'not_equal' with 'cond' input?

Reply:
Hi, we use this OP spec as its reference implementation
https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/not_equal_en.html#not-equal

Reply:
Got it. What about the question 1?

Reply:
This OP's return format is same as other elewise OPs, (e.g Equal), if Equal works, it should work as well.
https://www.paddlepaddle.org.cn/documentation/docs/en/develop/api/paddle/equal_en.html

Reply:
@meiyang-intel any suggestion?